.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: 'release'
+on:
+  push:
+    tags:
+      - 'v*'
+env:
+  # .goreleaser.docker.yaml reads CONTAINER_REGISTRY
+  CONTAINER_REGISTRY: 'us-docker.pkg.dev/abcxyz-artifacts/docker-images'
+  WIF_PROVIDER: 'projects/1096923323432/locations/global/workloadIdentityPools/github-pool/providers/github-provider'
+  WIF_SERVICE_ACCOUNT: 'gh-access-sa@jvs-ci-test.iam.gserviceaccount.com'
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
```

Comment:
I'm not sure we want to cancel in-progress releases? Seems bad to have something half-released.